packages/@okta/i18n/package.json

```diff
@@ -28,8 +28,8 @@
     "debug": "2.6.9",
     "decamelize": "1.2.0",
     "domelementtype": "1.3.0",
-    "domhandler": "2.3.0",
-    "domutils": "1.5.1",
+    "domhandler": "2.4.1",
+    "domutils": "1.6.2",
```

Comment:
Are we okay with updating these (and the one on line 59)? Doesn't look related to i18n.

Reply:
We are okay, checked with @mauriciocastillosilva-okta